DelvUI/Interface/HudManager.cs

```diff
@@ -283,9 +285,18 @@ protected unsafe bool ShouldBeVisible()
         {
             if (!ConfigurationManager.GetInstance().ShowHUD || Plugin.ClientState.LocalPlayer == null)
             {
+                _helper = null;
                 return false;
             }
 
+            if (_helper == null)
+            {
+                _helper = new HudHelper();
+                _helper.Configure(true, true);
```

Comment:
Whats the reason behind having to re-instantiate the helper every time the hud is hidden or the player reference is lost?

Reply:
Logging out of the game and back in with the same character, leads to loss of state in a sense, during first login I was configuring the HudHelper in UpdateJobs() if you properly load a job.

However on subsequent logins with the same character the current job hud is already loaded, HudManager - was never released and so UpdateJobs() method exits at the first If-statement, and that leads to for example combat hotbars showing up, or the ingame job gauge not being hidden, etc.